cobspec/src/main/java/com/saralein/cobspec/controller/AuthController.java

```diff
@@ -0,0 +1,58 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.server.controller.Controller;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+import com.saralein.server.response.ResponseBuilder;
+import java.util.Base64;
+
+public class AuthController implements Controller {
```

Comment:
This doesn't quite feel right to me. I think we are missing something somewhere. It could even be the that the `Controller` interface is misnamed. 

`AuthController` doesn't truly feel like it should be a `Controller`, know what I mean?